tests/factories/TractionPacbioSamplesFactory.js

```diff
@@ -0,0 +1,208 @@
+          },
+        },
+        {
+          id: '8',
+          type: 'requests',
+          attributes: {
+            library_type: 'null mock',
+            estimate_of_gb_required: 0,
+            number_of_smrt_cells: 0,
+            cost_code: 'nullmock',
+            external_study_id: '5470dfba-44ad-11e9-94c0-68b59977951e-mock',
+            sample_name: '5200STDY7323048mock',
+            barcode: 'TRAC-92',
+            sample_species: 'mock sample_common_name',
+            source_identifier: 'NT127Q',
+            created_at: '10/16/2019 10:00',
+          },
+        },
+        {
+          id: '9',
+          type: 'requests',
+          attributes: {
+            library_type: 'null mock',
+            estimate_of_gb_required: 0,
+            number_of_smrt_cells: 0,
+            cost_code: 'nullmock',
+            external_study_id: '5470dfba-44ad-11e9-94c0-68b59977951e-mock',
+            sample_name: '5200STDY7323048mock',
+            barcode: 'TRAC-93',
+            sample_species: 'mock sample_common_name',
+            source_identifier: 'NT127Q',
+            created_at: '10/16/2019 10:01',
+          },
+        },
+        {
+          id: '10',
+          type: 'requests',
+          attributes: {
+            library_type: 'null mock',
+            estimate_of_gb_required: 0,
+            number_of_smrt_cells: 0,
+            cost_code: 'nullmock',
+            external_study_id: '5470dfba-44ad-11e9-94c0-68b59977951e-mock',
+            sample_name: '5200STDY7323048mock',
+            barcode: 'TRAC-94',
+            sample_species: 'mock sample_common_name',
+            source_identifier: 'NT127Q',
+            created_at: '10/16/2019 10:03',
+          },
+        },
+        {
+          id: '11',
+          type: 'requests',
+          attributes: {
+            library_type: 'null mock',
+            estimate_of_gb_required: 0,
+            number_of_smrt_cells: 0,
+            cost_code: 'nullmock',
+            external_study_id: '5470dfba-44ad-11e9-94c0-68b59977951e-mock',
+            sample_name: '5200STDY7323048mock',
+            barcode: 'TRAC-95',
+            sample_species: 'mock sample_common_name',
+            source_identifier: 'NT127Q',
+            created_at: '10/16/2019 10:04',
+          },
+        },
+        {
+          id: '12',
+          type: 'requests',
+          attributes: {
+            library_type: 'null1x',
+            estimate_of_gb_required: 0,
+            number_of_smrt_cells: 0,
+            cost_code: 'null1x',
+            external_study_id: '5470dfba-44ad-11e9-94c0-68b59977951e',
+            sample_name: '5200STDY73230482x',
+            barcode: 'TRAC-96',
+            sample_species: 'Homo sapiens',
+            source_identifier: 'NT127Q',
+            created_at: '10/16/2019 10:10',
+          },
+        },
+      ],
+    },
+    status: 200,
+    statusText: 'Success',
+  }
+
+  return BaseFactory(data)
+}
+
+export default TractionPacbioSamplesFactory
```

Comment:
I believe that for all other existing factories, we start with the pipeline name. Would it be better to use `PacbioSamplesFactory` instead of `TractionPacbioSamplesFactory` to maintain uniformity and follow the same naming convention?